docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -134,6 +134,11 @@ a|Defines whether the tests run on real device
 !`mobile_app/android`
 !`false`
 
+|`mobile-environment.wda.bundle-id`
+|`string`
+|Depends on the set profile
+|The iOS platform property that defines the WebDriverAgent bundle ID installed on a iOS real device, in case of cloud testing the value can be found in the XCode logs
```

Comment:
please provide details what it's used for

Reply:
it's related to iOS security, an app in background can't access clipboard, so do end users really need to know what it's used for?

Reply:
no, I meant something like `The WDA bundle ID is used to retrieve clipboard text ...`

Reply:
doesn't it raise more questions? like why do we need bundle id to get clipboard text?